scripts/crowbar-testbuild.py

```diff
@@ -38,8 +38,14 @@
 }
 
 JOB_PARAMETERS = {
+    'crowbar-ha': ('nodenumber=4', 'hacloud=1',
+                   'networkingplugin=vxlan',
```

Comment:
Same question as @aspiers (but now for HA): Why vxlan? Is it just for testing it in one of the possible PRs so you decided to use the HA PRs to include vxlan?